### PR TITLE
[v10] chore: Bump Go to 1.19.7

### PR DESCRIPTION
**Pull request by codingllama**
Update Go to latest patch.

https://go.dev/doc/devel/release#go1.19.7

### CODE REVIEW

**Comment by codingllama** [comment]
Requires https://github.com/gravitational/teleport.e/pull/970. I'll fix this as soon as it lands.

**Review by mdwn — APPROVED** [review]
LGTM once the `e` change is in.

**Review by zmb3 — APPROVED** [review]

**Pull request merged**
